impl/client/src/main/javascript/web/util/_a11y.js

```diff
@@ -84,12 +81,34 @@ define(function () {
     }
   }
 
+  function createDisposable(dispose) {
+    dispose.remove = dispose;
+
+    return dispose;
+  }
+
+  function createNullDisposable() {
+    return createDisposable(function nullDispose() {
+      // NOOP
+    });
+  }
+
   return {
     keyCodes: keyCodes,
+
     makeAccessibleActionButton: function (elem) {
+      if (elem == null) {
+        return createNullDisposable();
+      }
```

Comment:
Ease of use comes with also having to return a disposable, even though a dummy one.